MobileApplication/MobileApplication/ViewModels/Goal/GoalsViewModel.cs

```diff
@@ -37,13 +43,35 @@ public GoalsViewModel()
         async Task ExecuteLoadItemsCommand()
         {
             IsBusy = true;
-            Items.Clear();
+            GoalViews.Clear();
 
             try
             {
-                var items = await RestService.RefreshDataAsync();
+                _goals = await RestService.RefreshDataAsync();
+                _transactions = await TransactionRestService.RefreshDataAsync();
+
+                _goals.ForEach(goal =>{
+                    decimal savePerMonth;
+                    decimal profitPerMonth;
+                    DateTime DeadlineDate = (DateTime)goal.Deadlinedate;
+                    decimal months = DeadlineDate.Subtract(goal.Creationdate).Days / 31;
+                    if (months > 1)
+                    {
+                        savePerMonth = goal.Amount / months;
+                    }
+                    else
+                    {
+                        savePerMonth = goal.Amount;
+                    }
+
+                    
+                    var goalView = new GoalView(goal);
+                    profitPerMonth = CalculateProfitPerMonth();
+                    goalView.Possibility = GoalPossibility(savePerMonth, profitPerMonth);
+                    goalView.MonthlyGoalAmount = Math.Round(savePerMonth,2);
```

Comment:
You could better use [`decimal.Divide(dec, dec)`](https://docs.microsoft.com/en-us/dotnet/api/system.decimal.divide?view=net-5.0) in line 60, and get rid of rounding.

Reply:
How would using Divide get rid of rounding?

Reply:
It returns a Decimal, not a float or double, that way you do not have to round it up.

Reply:
But it still shows a lot of digits after the decimal point without rounding